Add toolbar and headline section

diwakarstha opened the pull request:
Create portfolio page
- add index.html file
- add style.css file
- add toolbar section in index.html
- add headline section in index.html

diwakarstha committed:
setup linters for html and css

diwakarstha committed:
add index.html file

diwakarstha committed:
Add style.css file

diwakarstha committed:
create header on index.html

diwakarstha committed:
create headline section

diwakarstha committed:
Update README.md

-add screenshot
-add demo link

diwakarstha committed:
Update index.html

diwakarstha committed:
Update style.css

VICTOR-CHUKWU commented on index.html:
```diff
@@ -0,0 +1,40 @@
+          <li>Contact</li>
+          <li class="mail">
+            <img src="images/mail.png" alt="contact" />
+          </li> -->
```
- [x] Kindly remove commented codes as commented code makes our code base unreadable and not clean

VICTOR-CHUKWU commented on index.html:
```diff
@@ -0,0 +1,40 @@
+    <div class="wrapper">
+      <header class="nav">
+        <ul>
+          <li class="profile-name">Diwakar Shrestha</li>
```
- [ ] This is supposed to be the logo according to project design, kindly use the `<a>` tag as it is being required in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md). making your logo a `<a>` tag is good for semantic reasons

VICTOR-CHUKWU commented on index.html:
```diff
@@ -0,0 +1,40 @@
+          <li class="mail">
+            <img src="images/mail.png" alt="contact" />
+          </li> -->
+          <li><img src="images/toggle.png" alt="toggle-bar" /></li>
```
- [ ] According to [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), kindly wrap the toggle bar in `nav` tag

VICTOR-CHUKWU commented on index.html:
```diff
@@ -0,0 +1,40 @@
+          </li> -->
+          <li><img src="images/toggle.png" alt="toggle-bar" /></li>
+        </ul>
+      </header>
```
- [ ] It is a good practice for a frontend developer to follow the Figma design given. Your design is near perfection, I think it is better to have a space between the logo and toggle bar. kindly follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346)
Figma design 👇 
                        
![image](https://user-images.githubusercontent.com/66097371/148257242-97c4b775-f581-4b36-87b8-3a6782b798cc.png)

Your design 👇 
                  
![image](https://user-images.githubusercontent.com/66097371/148257397-1e970d5f-7424-4e04-ab43-d68901e3bbea.png)

VICTOR-CHUKWU commented on index.html:
```diff
@@ -0,0 +1,40 @@
+      </header>
+
+      <section class="headline">
+        <p class="name"><strong>Diwakar Shrestha</strong></p>
```
- [ ] kindly change the `p` tag to `h1` tag as requested in the project requirement. `h1` as the main title is good for semantic and SEO reasons. kindly check this [HTML and CSS best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

VICTOR-CHUKWU commented on index.html:
```diff
@@ -0,0 +1,40 @@
+
+      <section class="headline">
+        <p class="name"><strong>Diwakar Shrestha</strong></p>
+        <p class="position">Software Engineer</p>
```
- [ ] Also according to project requirement kindly change this subheading to `heading` tag i.e `h2` or `h3`. check this [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

VICTOR-CHUKWU commented on style.css:
```diff
@@ -0,0 +1,71 @@
+  flex-wrap: wrap;
+}
+
+/*
+  Header
+*/
```
- [ ] Just as mentioned above, It would be nice if we keep our code base clean by removing commented codes

VICTOR-CHUKWU commented on style.css:
```diff
@@ -0,0 +1,71 @@
+  flex: 1;
+}
+
+/*
+  Headline Section
+*/
```
- [ ] Kindly remove comments as code, you did a nice job naming your class in a descriptive way, therefore is no need for the comments

VICTOR-CHUKWU reviewed (changes requested):
Hi @diwakarstha  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![almost](https://media.giphy.com/media/R2m2NzVxQ3pbG/giphy.gif)

## Highlights
- Good PR
- Good commit message
- Descriptive README file
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

diwakarstha committed:
Update index.html file

- remove commented codes
- use <a> tag to wrap logo
- wrap toggle bar in nav
- replace p tag with heading tag

diwakarstha committed:
Update styles in style.css 

- change .nav to header
- remove commented codes

diwakarstha committed:
Update index.html

diwakarstha committed:
Update layout index.html and style.css

diwakarstha committed:
Add background image style.css

diwakarstha committed:
Remove pixel on margin

diwakarstha committed:
Update README.md

Cathella reviewed (approved):
## 🟢 Status: Approved 🎉 💯 👏🏻 

### Hi @diwakarstha 👋🏻 , 

Congratulations dear! You did a great job making changes requested by the previous reviewer 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
